static/app/views/insights/browser/webVitals/components/charts/performanceScoreChart.tsx

```diff
@@ -33,10 +28,7 @@ export const ORDER: WebVitals[] = ['lcp', 'fcp', 'inp', 'cls', 'ttfb'];
 export function PerformanceScoreChart({
   projectScore,
   webVital,
-  transaction,
```

Comment:
only `pageOverview` passes `transaction`, and it comes from query params, so this was moved into the new widget